OpenLaTeXStudio/EditorModule/nbproject/project.xml

```diff
@@ -105,6 +105,14 @@
                     </run-dependency>
                 </dependency>
                 <dependency>
+                    <code-name-base>org.openide.util.ui</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>9.4.1</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>

```

Comment:
I'm getting the following when attempting to build the project with those changes, which I assume is related with this new dependency:

```
C:\Program Files\NetBeans 8.0.2\harness\suite.xml:187: The following error occurred while executing this line:
C:\Program Files\NetBeans 8.0.2\harness\build.xml:174: No dependent module org.openide.util.ui
BUILD FAILED (total time: 2 seconds)
```